src/web/handlers.rs

```diff
@@ -68,16 +62,19 @@ pub fn get_quota(meta: MetaRequest) -> FutureResponse<HttpResponse> {
     )
 }
 
-pub fn delete_all(meta: MetaRequest) -> FutureResponse<HttpResponse> {
+pub fn delete_all(meta: MetaRequest) -> impl Future<Item = HttpResponse, Error = Error> {
+    #![allow(clippy::unit_arg)]
     Box::new(
```

Comment:
Same here

Reply:
good catch, we probably just forgot these